aws/tf/modules/sra/databricks_workspace/workspace_security_modules/uc_catalog/uc_catalog.tf

```diff
@@ -0,0 +1,164 @@
+resource "null_resource" "previous" {}
```

Comment:
Minor nit...with a goal of making modules as unaware as possible, perhaps move this sleep operation to the root module?

Reply:
since this occurs at the module level, I think I'm going to leave for now since the race condition comes with the IAM role and the external location